lib/Parser/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ add_library(FortranParser
 )
 
 target_link_libraries(FortranParser
-  FortranCommon
+  FortranCommon LLVMSupport
```

Comment:
General style seems to be one entry per library/file.

Reply:
Done!